src/temp/TimeTravelDropdown/TimeTravelDropdown.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useState } from 'react';
+import styles from './Dropdown.module.scss';
+import CalendarIcon from '../icons/CalendarIcon';
+import DropdownArrow from '../icons/DropdownArrow';
+
+interface TimeTravelDropdownProps {
+  labelYear: string;
+  labelSource: string;
+  yearList: string[];
+  sourceList: string[];
+  isOpen: boolean;
+}
+
+const TimeTravelDropdown = ({
+  labelYear,
+  labelSource,
+  yearList,
+  sourceList,
+  isOpen,
+}: TimeTravelDropdownProps) => {
+  const [selectedYear, setSelectedYear] = useState(labelYear);
+  const [selectedSource, setSelectedSource] = useState(labelSource);
+  const [isMenuOpen, setIsMenuOpen] = useState(isOpen);
+  const handleChangeYear = (year: string) => {
+    setSelectedYear(year);
+  };
+
+  const handleChangeSource = (source: string) => {
+    setSelectedSource(source);
+  };
+
+  const styleForSelectedOption = (
+    selectedOption: string,
+    labelValue: string
+  ) => {
+    if (selectedOption.toLowerCase() === labelValue.toLowerCase()) {
+      return {
+        fontWeight: 700,
+        color: `${'var(--selected-menu-item-color-new)'}`,
+      };
+    } else {
+      return undefined;
+    }
+  };
+
+  return (
+    <div className={styles.menuContainer}>
+      <div
+        className={styles.menuButton}
+        onClick={() => setIsMenuOpen(!isMenuOpen)}
+      >
```

Comment:
Using a `<button>` element instead of a` <div>` would be more semantically appropriate for handling click events